geoportal/vars.yaml

```diff
@@ -74,6 +74,10 @@ vars:
           params:
             limit: 30
             partitionlimit: 5
+        ngeoPermalinkOgcserverUrl:
+          name: mapserverproxy
+          params:
+            ogcserver: 'QGIS server'
```

Comment:
do we only have qgis-server layers in the demo? or is this just set as an example ?
what happen if we want wfspermalink in a project using both mapserver and qgisserver ?

Reply:
It's not possible to create WFS permalink link one different OGC servers :-/

Reply:
ok, thank you